src/content/en/updates/2017/12/chrome-loadtimes-deprecated.md

```diff
@@ -0,0 +1,358 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The non-standard chrome.loadTimes() API will be deprecated in Chrome 64 now that standards-based equivalents exist for all of its features.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: performance #}
+{# wf_blink_components: Blink>PerformanceAPIs #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: The non-standard chrome.loadTimes() API will be deprecated in Chrome 64 now that standards-based equivalents exist for all of its features. #}
+{# wf_blink_components: Blink>PerformanceAPIs #}
+
+# Chrome 64 to deprecate the chrome.loadTimes() API {: .page-title }
+
+{% include "web/_shared/contributors/philipwalton.html" %}
+
+`chrome.loadTimes()` is a non-standard API that exposes loading metrics and
+network information to developers in order to help them better understand their
+site's performance in the real world.
+
+Since this API was implemented in 2009, all of the information it reports can be
+found in standardized APIs such as [Navigation Timing 2](https://www.w3.org/TR
```

Comment:
"all of the information it reports can be found in standardized APIs". I thought that wasn't 100% true? :)

Reply:
Yeah, the stuff we haven't replace applies to deprecated or outdated features. I'll reword to "all the *useful* information..."